lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -864,26 +864,26 @@ export async function updatePr({
       throw Object.assign(new Error(REPOSITORY_NOT_FOUND), { statusCode: 404 });
     }
 
+    const body: HttpOptions['body'] = {
```

Comment:
```suggestion
    const body: any = {
```
I'm pretty sure that the same as any. we should create an interface for this